src/components/schedule-page/table/schedule/schedule-parts/schedule-folding-section.component.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { ReactNode } from "react";
+import { useState } from "react";
+import ArrowRightIcon from "@material-ui/icons/ArrowRight";
+import ArrowDropDownIcon from "@material-ui/icons/ArrowDropDown";
+interface ScheduleFoldingSectionOptions {
+  name: string;
+  children: ReactNode;
+}
+/**
+ * Used only in schedule. For other use cases, use @see FoldingSection from  @module common-components
+ */
+export function ScheduleFoldingSection({
```

Comment:
Osobna zwijalna sekcja która jest używana wyłącznie w `ScheduleComponent` w związku z tym, że ten komponent jest zrobiony jako tabelka, przez co on potrzebuje mieć specyficzny layout. 
Może nie jest to najładniejsze rozwiązanie, ale nie myslę, że ten komponent, będzie się jakoś zmieniać, więc takie rozwiązanie może być.